### DISCUSSION TITLE
Vulnerability scan

### DISCUSSION THREAD

**Discussion by CISO-GYM** [wazuh/wazuh#22273] [General]
I did all the steps of vulnerability scanning and made sure that the cve.db is downloaded but when i go into the dashboard to check it the vulnerabilities are empty

**Comment by MiguelazoDS** [comment]
Hi @IRobinHood, 

- What agent OS are you trying to scan? 
- Are the agents connected and active? 
- Can you see in the dashboard the last scan timestamp?

**Comment by v-milhem** [comment]
Same here, but only for Windows 11 agent. Haven't tried other Windows versions, but Ubuntu 20.04.6 and macOS 13.6.4 work.

**Reply by MiguelazoDS** [reply]
Hi @v-milhem 
If we have a fully patched Windows agent, we can expect to not have any vulnerability related to that OS, but sometimes what could be happening is the lack of translation for certain installed packages. https://documentation.wazuh.com/current/user-manual/capabilities/vulnerability-detection/cpe-helper.html. We are changing that behavior to avoid manually editing the dictionary to add new entries.
